src/logging.cc

```diff
@@ -1224,10 +1226,12 @@ void LogFileObject::Write(bool force_flush,
     if(!g_application_fingerprint.empty()) {
       file_header_stream << "Application fingerprint: " << g_application_fingerprint << '\n';
     }
-
+    string date_time_format = FLAGS_log_year_in_prefix
```

Comment:
Nit: `std::string` is not really necessary here, is it? My guess `const char*` would be sufficient. Something along the lines:
```suggestion
    const char* const date_time_format = FLAGS_log_year_in_prefix
```